hammer/hammer.go

```diff
@@ -103,6 +104,7 @@ func NewHammer(tracker *client.LogStateTracker, f client.Fetcher, w LeafWriter)
 	readThrottle := NewThrottle(*maxReadOpsPerSecond)
 	writeThrottle := NewThrottle(*maxWriteOpsPerSecond)
 	errChan := make(chan error, 20)
+	leafSampleChan := make(chan leafTime, 100)
```

Comment:
Maybe `leafWriteStats` instead? It isn't sampled from what I can see, and it only comes from the writer workers.

Reply:
well, it's sampled in a very primitive sense in that the channel only takes 100 entries, and if it's full we just drop the sample.

Happy to change that approach though.